tests/new_upgrades/test_errata.py

```diff
@@ -0,0 +1,285 @@
+        custom_yum_repo = target_sat.api.Repository(
+            product=product.id,
+            name=f'{test_name}_repo',
+            content_type='yum',
+            url=settings.repos.yum_9.url,
+        ).create()
+        test_data.custom_repo_id = custom_yum_repo.id
+        product.sync()
+        synced_repos = _create_custom_rhel_n_client_repos(
+            target_sat, product, rhel_contenthost.os_version.major
+        )
+        test_data.synced_repo_ids = [repo.id for repo in synced_repos]
+        repolist = [
+            custom_yum_repo,
+            *synced_repos,
+        ]
+        content_view = target_sat.publish_content_view(org, repolist, f'{test_name}_cv')
+        test_data.content_view_id = content_view.id
+        ak = target_sat.api.ActivationKey(
+            name=f'{test_name}_ak',
+            content_view=content_view,
+            organization=org.id,
+            environment=environment,
+        ).create()
+        test_data.activation_key = ak.name
+        if not target_sat.is_sca_mode_enabled(org.id):
+            subscription = target_sat.api.Subscription(organization=org).search(
+                query={'search': f'name={product.name}'}
+            )[0]
+            ak.add_subscriptions(data={'subscription_id': subscription.id})
+        # Override/enable all AK repos (disabled by default since 6.15)
+        c_labels = [
+            i['label'] for i in ak.product_content(data={'content_access_mode_all': '1'})['results']
+        ]
+        ak.content_override(
+            data={
+                'content_overrides': [{'content_label': label, 'value': '1'} for label in c_labels]
+            }
+        )
+        rhel_contenthost.api_register(target_sat, organization=org, activation_keys=[ak.name])
+        rhel_contenthost.add_rex_key(satellite=target_sat)
+        rhel_contenthost.install_katello_host_tools()
+        rhel_contenthost.execute('subscription-manager refresh')
+
+        _check_yum_plugins_count(rhel_contenthost)
+
+        pkg_install = rhel_contenthost.execute(
+            f'yum -y install {" ".join(constants.FAKE_9_YUM_OUTDATED_PACKAGES)}'
+        )
+        assert pkg_install.status == 0, 'Failed to install packages'
+        host = target_sat.api.Host().search(query={'search': f'activation_key={ak.name}'})[0]
+        assert host.id == rhel_contenthost.nailgun_host.id, 'Host not found in Satellite'
+        assert rhel_contenthost.applicable_errata_count > 1, (
+            f'No applicable errata found for host {host.name}'
+        )
+        erratum_list = target_sat.api.Errata(repository=custom_yum_repo).search(
+            query={'order': 'updated ASC', 'per_page': 1000}
+        )
+        errata_ids = [errata.errata_id for errata in erratum_list]
+        assert sorted(errata_ids) == sorted(settings.repos.yum_9.errata)
+        sat_upgrade.ready()
+        target_sat._session = None
+        yield test_data
+
+
+@pytest.mark.rhel_ver_list([7, 8, 9, 10])
```

Comment:
Could we use an implicit parametrization of `rhel_contenthost`, 
to avoid having to update this marker, for each RHEL added/dropped from support?

Using `rhel_ver_match` format 'N-x' (newest rhel N, and `x` prior maj versions, no fips) :
```suggestion
@pytest.mark.rhel_ver_match('N-3')
```
Or with regex pattern (all major versions, no fips):
```suggestion
@pytest.mark.rhel_ver_match(r'^(?!.*fips).*$')
```
Or keeping use of `rhel_ver_list`, something along these lines? (all major versions, no fips) :
```suggestion 
@pytest.mark.rhel_ver_list(
    [
         str(r) 
         for r in settings.supportability.content_hosts.rhel.versions
         if isinstance(r, int)
     ]
)
```

Reply:
@damoore044 Good call. I just tested your regex match locally and the tests passed, so I will go with that option.

Reply:
Nice ack, keeping in mind in 6.16 and 6.15 CPs, I think we should just 
expect 3 params instead; [rhel7, 8, 9]